bokehjs/src/coffee/models/ranges/data_range1d.coffee

```diff
@@ -22,7 +22,7 @@ export class DataRange1d extends DataRange
   }
 
   @internal {
-      mapper_hint:     [ p.String, 'auto' ]
+      scale_hint:     [ p.String, 'auto' ]
```

Comment:
Is the intention to remove the scale hinting (and pass first class models) in a follow-up PR?